pkg/binding/binding.go

```diff
@@ -38,7 +38,7 @@ func (c *Controller) syncBinding(ctx context.Context, bindingName string) error
 		// then the bindingpolicy has been deleted, and the binding
 		// will eventually be garbage collected. We can safely ignore this.
 
-		c.bindingPolicyResolver.Broker().NotifyCallbacks(bindingName)
+		c.bindingPolicyResolver.Broker().NotifyBindingPolicyCallbacks(bindingName)
```

Comment:
Who notifies singleton-status subscribers if this possibly newly recorded absence affects any?